### PR TITLE
Fix/gather timeout

### PR DESCRIPTION
**Pull request by xquanluu**
Fix gather cannot be timeout if missing `minDigits`

https://github.com/jambonz/jambonz-feature-server/issues/597

### CODE REVIEW

**Comment by davehorton** [comment]
can you open an issue and describe this bug?

**Comment by xquanluu** [comment]
Opened issue

File: lib/tasks/gather.js [hunk 1]
```diff
@@ -35,6 +35,7 @@ class TaskGather extends SttTask {
 
     /* timeout of zero means no timeout */
     this.timeout = this.timeout === 0 ? 0 : (this.timeout || 15) * 1000;
+    this.interDigitTimeout = (this.interDigitTimeout > 0 ? this.interDigitTimeout : 0) * 1000;
```
Comment (davehorton) [inline lib/tasks/gather.js:38]: was this changed needed to fix this bug?  Or is it an unrelated fix?

Comment (xquanluu) [reply lib/tasks/gather.js]: this is to make sure there is value for interDigitTimeout that later we can do compare this. interDigitTimeout <= 0.

But I think this is not needed.

I added new changes and retested on jambonz.me

Comment (davehorton) [reply lib/tasks/gather.js]: Yes I have a feeling we are over-complicating things here.  I will have more time to review later today

**Review by davehorton — APPROVED** [review]

File: lib/tasks/gather.js (hunk 1 above)
Comment (davehorton) [reply lib/tasks/gather.js]: where are we with this PR @xquanluu ?

**Comment by xquanluu** [comment]
I remembered that you gave me some suggestion to solve this problem, I applied and retested on jambonz.me, and it was working at that time.

It's good to be merged

**Review by davehorton — APPROVED** [review]
